rx-usecases-test/src/main/java/com/thefuntasty/mvvm/rxusecases/test/CompletableUseCase.kt

```diff
@@ -0,0 +1,76 @@
+package com.thefuntasty.mvvm.rxusecases.test
+
+import com.thefuntasty.mvvm.rxusecases.disposables.CompletableDisposablesOwner
+import com.thefuntasty.mvvm.rxusecases.usecases.CompletableUseCase
+import io.mockk.every
+import io.mockk.just
+import io.mockk.runs
+import io.reactivex.Completable
+import io.reactivex.disposables.Disposable
+
+/**
+ * Mock [CompletableDisposablesOwner.execute] method.
+ *
+ * When the execute method will be called then the argument passed in [resultBlock] will be used as a result of mocked use case
+ * and corresponding methods for the given use case will be called.
+ * So when `Completable.complete` will be passed then `onComplete` will be called etc.
+ *
+ * Usage:
+ * mockCompletableUseCase.everyExecute(args = ...) { Completable.complete() }
+ */
+inline fun <reified ARGS : Any, USE_CASE : CompletableUseCase<ARGS>> USE_CASE.everyExecute(args: ARGS, resultBlock: () -> Completable) {
```

Comment:
Similarly to my previous comment, I think it is not important to name this method after the fact that you are mocking every execute, but what the result of this action is -> `mockExecutedCompletable`, `fakeExecutedCompletable`, `replaceExecutedCompletable`,  ...

Reply:
I don't think that `Completable` suffix is needed. When you will return different parameter then an error message is pretty clear.

![image](https://user-images.githubusercontent.com/6089152/72065635-facba500-32de-11ea-9a0b-9dac5857b157.png)

`mockUseCase.mockExecuteMaybe { Maybe.just(expectedResult) }` 

`Maybe` is used two times subsequentially. That is unnecessary I would say.

Reply:
Also, you need to think about nullable use cases. I that case it would look like this:
`mockUseCase.mockExecuteMaybeNullable { Maybe.just(expectedResult) }`
and it looks pretty ugly to me.

 I would change it to `mockExecute`

Reply:
I'm fine with `mockExecute`.